compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -62,7 +62,7 @@ luci::CircleQuantize *create_quantize_op(luci::CircleNode *node, loco::DataType
   auto qtype = luci::activation_qtype(node);
   if (qtype == ActivationQType::PreDefinedValue)
   {
-    quantize->quantparam(luci::make_predefined_qparam(node->opcode(), out_type));
+    quantize->quantparam(luci::make_predefined_qparam(node, out_type));
```

Comment:
For reviewers:

This PR allows to set the pre-defined qparam of a new Quantize OP using the input of Quantize Op.